readme.md

```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://KhomichYehor.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://KhomichYehor.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="933" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/9a7163bd-602f-405e-8539-9ed78d611d24">